mana_decay_provider_test.go

```diff
@@ -16,6 +16,7 @@ const (
 	betaPerYear                  float64 = 1 / 3.0
 	slotsPerEpochExponent                = 13
 	slotDurationSeconds                  = 10
+	bitsExponent                         = 63
```

Comment:
```suggestion
	bitsCount                         = 63
```

Anything with "exponent" should be removed. That was incorrect before, my bad.